deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/config/GithubFactory.java

```diff
@@ -37,6 +37,10 @@ public class GithubFactory implements SourceControlFactory {
     @JsonProperty
     private String appOrganization;
 
+    @NotNull
+    @JsonProperty
+    private String typeName;
```

Comment:
Yes, when a GithubManager or PhabricatorManager object is instantiated it will need to have access to this Json value from the config file since it holds a specific value

Reply:
Missed that part, thanks!